react-accordion/my-app/src/TopicCard.tsx

```diff
@@ -0,0 +1,20 @@
+type Props = {
+  topics: { id: number; title: string; content: string }[];
```

Comment:
Ideally, a TopicCard would display just one Topic. That makes it nice and simple, and easy to reuse.